packages/view-types/statistical-plots/src/CellSetExpressionPlot.js

```diff
@@ -66,6 +66,8 @@ export default function CellSetExpressionPlot(props) {
 
   const rectColor = (theme === 'dark' ? 'white' : 'black');
 
+  const captializedObsType = capitalize(featureValueType);
```

Comment:
What is the motivation for this change? It changes the X-axis title from Cell Set to Expression Set which I do not think is needed

![Screen Shot 2023-04-05 at 2 53 58 PM](https://user-images.githubusercontent.com/7525285/230177798-41b75dbc-e73e-4c6f-a7e7-b1438796060b.png)